tribe-ext-custom-all-events-url.php

```diff
@@ -113,7 +161,8 @@ public function add_settings() {
 		 * @return mixed
 		 */
 		function custom_all_events_url( $url ) {
-			$custom_url = tribe_get_option( $this->opts_prefix . 'custom_all_events_url' );
+
+			$custom_url = $this->get_custom_url();
```

Comment:
```suggestion

			$custom_url = $this->get_settings()->get_option( 'custom_all_events_url' );
```

Reply:
Was outdated, couldn't commit, so changed it.